server/hedley/modules/custom/hedley_person/scripts/migrate-children.php

```diff
@@ -22,7 +22,7 @@
 $batch = drush_get_option('batch', 50);
 
 // Get allowed memory limit.
-$memory_limit = drush_get_option('memory_limit', 500);
+$memory_limit = drush_get_option('memory_limit', 800);
```

Comment:
Let's verify if DDEV has the same (elevated) memory limit like the new Pantheon limit.
https://github.com/Gizra/drupal-starter/blob/main/.ddev/php/memory-limit.php.ini

Reply:
Done